code/game/objects/structures/holosign.dm

```diff
@@ -101,7 +102,7 @@
 
 /obj/structure/holosign/barrier/cyborg/hacked
 	name = "Charged Energy Field"
-	desc = "A powerful energy field that blocks movement. Energy arcs off it."
+	desc = "A powerful energy field that blocks movement. Energy arcs off of it."
```

Comment:
Just off is fine here, see https://grammarist.com/usage/off-of/
At that point it's a stylistic choice, and I don't see why we should go for the more casual, spoken variant here.